migrations/2022/Version20220916214048.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
Please use a single migration for every ticket / feature and remove the faulty code snippets

Pretty sure you haven't changed studio - that's a bug in the generation with enums :/